e2e/integration/ssitests.js

```diff
@@ -1,24 +1,55 @@
 import { apps } from "../helper/apps.list";
+import Sidebar from "../pageObject/Sidebar"
 context( 'Test for SSI', () => {
-
+    const sidebar = new Sidebar();
     before( () => {
-        cy.viewport(Cypress.env('width'), Cypress.env('height') );
-        cy.visit( Cypress.env( 'QA_HOST' ));
+        cy.viewport( Cypress.env( 'width' ), Cypress.env( 'height' ) );
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
         cy.login( Cypress.env( 'USERNAME' ), Cypress.env( 'PASSWORD' ) );
     } );
 
     it( 'Test for Notifications section', () => {
-        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click({force:true}).then( () => {
+        cy.wait(3000)
+        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click( { force: true } ).then( () => {
+
+            cy.get( '.opc-notification-drawer__header-title', { force: true } ).invoke( 'text' ).then( ( text ) => {
+                expect( text.trim() ).equal( 'Notifications' );
+            } );
+
+
+        } );
+        cy.get( ".opc-notification-drawer__header > button" ).click( { force: true } );
+    } );
+    it( 'Test Application Search in Side Bar', () => {
+        cy.get( 'slot[name="opc-nav-btn"] > button:last-child' ).focus().click( { force: true } ).then( () => {
+cy.wait(2000)
+            cy.get( '.opc-menu-drawer' ).within( () => {
 
-            cy.get( '.opc-notification-drawer__header-title',{force:true} ).invoke( 'text' ).then( ( text ) => {
-                expect( text.trim()).equal('Notifications' )
+                cy.get( 'input[name="seach"]' ).focus().type( "dsal" ,{force:true});
+                sidebar.getApplicationSearchResults().contains('DSAL' ).should( 'be.visible' );
```

Comment:
@mudit94 DSAL will not be always visible. The applications can change in the future

Reply:
hi @hybridx , i have fixed this .

Reply:
Hi @hybridx , please merge this 1 for now. I shall create different jira issues, different PR's from next time.:)

Reply:
https://github.com/1-Platform/one-platform/pull/1523/commits/801d90cc8905d15a2a9c057b2c62cd1817c67276